deploy/olm-catalog/windows-machine-config-operator/manifests/windows-machine-config-operator.clusterserviceversion.yaml

```diff
@@ -12,11 +17,126 @@ metadata:
 spec:
   apiservicedefinitions: {}
   customresourcedefinitions: {}
-  description: Placeholder description
+  description: |-
+    Issues with this distribution of WMCO can be opened against the [WMCO component Bugzilla](https://bugzilla.redhat.com/buglist.cgi?product=OpenShift%20Container%20Platform&component=Windows%20Containers&resolution=---).
+    Please read through the [troubleshooting document](https://github.com/openshift/windows-machine-config-operator/blob/community-4.6/docs/TROUBLESHOOTING.md)
+    before opening an issue.
```

Comment:
Please move this paragraph to the end under a `Reporting issues` section

Please replace `WMCO component Bugzilla` with `Windows Container Bugzilla component`. The link is needed.